processor/tailsamplingprocessor/README.md

```diff
@@ -204,6 +204,210 @@ processors:
 
 Refer to [tail_sampling_config.yaml](./testdata/tail_sampling_config.yaml) for detailed examples on using the processor.
 
+## A Practical Example
+
+Imagine that you wish to configure the processor to implement the following rules:
+
+1. **Rule 1:** Not all teams are ready to move to tail sampling. Therefore, sample all traces that are not from the team `team_a`.
+
+1. **Rule 2:** Sample only 1 percent of Readiness/liveness probes
+
+1. **Rule 3:** `service-1` has a noisy endpoint `/v1/name/{id}`. Sample only 1 percent of such traces.
+
+1. **Rule 4:** Other traces from `service-1` should be sampled at 100 percent.
+
+1. **Rule 5:** Sample all traces if there is an error in any span in the trace.
+
+1. **Rule 6:** Add an escape hatch. If there is an attribute called `app.force_sample` in the span, then sample the trace at 100 percent.
+
+Here is what the configuration would look like:
+
+```yaml
+tail_sampling:
+  decision_wait: 10s
+  num_traces: 100
+  expected_new_traces_per_sec: 10
+  policies: [
+      {
+        # Rule 1: use always_sample policy for services that are not using tail sampling
+        name: backwards-compatibility-policy,
+        type: and,
+        and:
+          {
+            and_sub_policy:
+              [
+                {
+                  name: services-using-tail_sampling-policy,
+                  type: string_attribute,
+                  string_attribute:
+                    {
+                      key: service.name,
+                      values:
+                        [
+                          list,
+                          of,
+                          services,
+                          using,
+                          tail_sampling,
+                        ],
+                      invert_match: true,
+                    },
+                },
+                { name: sample-all-policy, type: always_sample },
+              ],
+          },
+      },
+      # BEGIN: policies for team_a
+      {
+        # Rule 2: low sampling for readiness/liveness probes
+        name: team_a-probe,
+        type: and,
+        and:
+          {
+            and_sub_policy:
+              [
+                {
+                  # filter by service name
+                  name: service-name-policy,
+                  type: string_attribute,
+                  string_attribute:
+                    {
+                      key: service.name,
+                      values: [service-1, service-2, service-3],
```

Comment:
nit: since this is an example code, I would find it easier to read if the formatting of the policies matched. for example here the values in the array are listed in a single line whereas in line 245 they're split up across multiple lines.

Reply:
i let auto-formatters make such decisions - which ends up producing the most readable result. I believe the formatter automatically splits into separate lines them if the single line is above a certain threshold number of characters.